AFNetworking/AFURLRequestSerialization.m

```diff
@@ -847,11 +847,14 @@ @interface AFMultipartBodyStream () <NSCopying>
 @end
 
 @implementation AFMultipartBodyStream
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wimplicit-atomic-properties"
```

Comment:
Well, the issue stems from the declaration of the properties in NSStreamDelegate, where they're declared without the 'nonatomic' keyword. We can consider re-declaring them as nonatomic, but I'm not sure what would be the repercussions, or how to test for them.